temporal-normalizer/README.md

```diff
@@ -0,0 +1,72 @@
+# Illinois-time
+
+Illinois-time is a temporal extraction and normalization framework, designed to detect the temporal phrase of given text, and normalize them according to [TIMEX3 standard](http://www.timeml.org/tempeval2/tempeval2-trial/guidelines/timex3guidelines-072009.pdf)
+
+## Usage
+
+If you want to use this in your project, you need to take two steps. First add the dependencies, and then call the functions
+in your program.
+Here is how you can add maven dependencies into your program:
+
+```xml
+    <repositories>
+        <repository>
+            <id>CogCompSoftware</id>
+            <name>CogCompSoftware</name>
+            <url>http://cogcomp.cs.illinois.edu/m2repo/</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-time</artifactId>
+            <version>VERSION</version>
+        </dependency>
+    </dependencies>
+```
+
+**Note:** Make sure to change the pom.xml parameter `VERSION` to the latest version of the project.
+
+In general, the best way to use the illinois-time is through the [`TemporalChunkerAnnotator class`](src/main/java/edu/illinois/cs/cogcomp/temporal/normalizer/main/TemporalChunkerAnnotator.java). Like any other annotator, it is used by calling the `addView()` method on the `TextAnnotation` containing sentences to be tagged.
+
+```java
+	Properties rmProps = new TemporalChunkerConfigurator().getDefaultConfig().getProperties();
+	// Whether you want to use HeidelTime or our normalizer for normalization
+	rmProps.setProperty("useHeidelTime", useHeidelTime ? "True" : "False");
+	TemporalChunkerAnnotator tca = new TemporalChunkerAnnotator(new ResourceManager(rmProps));
+	tca.addView(ta);
+```
```

Comment:
this is great. But could you add more flesh to this example? 
Like how to create ta from raw text? Also let's include the imports, since not everyone is using IDEs. 
Here is an example from NER: 

```
import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
import edu.illinois.cs.cogcomp.nlp.utility.TokenizerTextAnnotationBuilder;
import edu.illinois.cs.cogcomp.annotation.TextAnnotationBuilder;
import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
import edu.illinois.cs.cogcomp.ner.NERAnnotator;
import edu.illinois.cs.cogcomp.nlp.tokenizer.StatefulTokenizer;
import edu.illinois.cs.cogcomp.ner.LbjTagger.*;
import java.io.IOException;

import java.util.Properties;

// Filename: App.java
public class App
{
    public static void main( String[] args ) throws IOException
    {
        String text1 = "Good afternoon, gentlemen. I am a HAL-9000 "
            + "computer. I was born in Urbana, Il. in 1992";

        String corpus = "2001_ODYSSEY";
        String textId = "001";

        // Create a TextAnnotation using the LBJ sentence splitter
        // and tokenizers.
        TextAnnotationBuilder tab;
        // don't split on hyphens, as NER models are trained this way
        boolean splitOnHyphens = false;
        tab = new TokenizerTextAnnotationBuilder(new StatefulTokenizer(splitOnHyphens));

        TextAnnotation ta = tab.createTextAnnotation(corpus, textId, text1);

        NERAnnotator co = new NERAnnotator(ViewNames.NER_CONLL);
        co.getView(ta);

        System.out.println(ta.getView(ViewNames.NER_CONLL));
    }
}
```